libvmi/driver/driver_interface.h

```diff
@@ -203,5 +209,9 @@ status_t driver_init_vmi(
     uint32_t init_flags,
     vmi_init_data_t *init_data);
 
+status_t driver_light_init(
```

Comment:
Could this not be folded into `driver_init`? I'm not a fan of introducing yet another obscure driver init function. It's already quite opaque having two separate ones.

Reply:
Yes, I can move that code at the end of driver_init

Reply:
On a second look that would involve to init xen_events_t * xe in xen.c and not in xen_events.c. If that is ok then I will move it

Reply:
Hm, no, that doesn't sound right. How about moving it into `xen_init_events`?

Reply:
There was a problem with that in the idea that xen_init_events() needs to have xen_init_vmi() called before(or it is called by) so that is why I opted for a different function because the code would have changed very much

Reply:
Right. In that case I guess just pick a better name for this, instead of driver_light_init make it driver_domainwatch_init

Reply:
I've changed the name and pushed